daemon/lokinet.cpp

```diff
@@ -34,11 +34,12 @@ namespace
     bool generate = false;
     bool router = false;
     bool force = false;
+    bool config = false;
     bool configOnly = false;
     bool overwrite = false;
 
     // string options
-    std::string configPath;
+    std::string configPath = "";
```

Comment:
this really should be an `fs::path`

`fs::path configPath{};`

Reply:
I was attempting to follow the logic of the previous implementation:
- Original: [the path is extracted from 'result' and then stored in the fs::path 3 lines later](https://github.com/oxen-io/lokinet/blob/bf0dc52df7c05d4a14b5d5140615c1d547430055/daemon/lokinet.cpp#L454)
- Updated: path is stored in struct as string, and if it is updated then it is stored into the [fs::path here](https://github.com/drouhana/lokinet/blob/06b2c58a7dc35d05fe250bc428a00c13209cf0e1/daemon/lokinet.cpp#L397)

Would you like me to change it so that the struct stores the fs::path right off the bat?

Reply:
I don't think we should change it because, while CLI11 supports std::filesystem::path, but our `fs` is sometimes `ghc::filesystem` which wouldn't work on those platforms.

So perhaps leave a comment along the lines of:
```C++
// TODO: change this to use a std::filesystem::path once we stop using ghc::filesystem on some platforms
```

Reply:
However the addition of the `= "";` initializer here is unnecessary: strings always initialize to an empty string.

Reply:
I only added the `= ""` so I could compare [here](https://github.com/drouhana/lokinet/blob/06b2c58a7dc35d05fe250bc428a00c13209cf0e1/daemon/lokinet.cpp#L426), as it gave me an error that the expression using a string has to evaluate to bool-type. I can fix that

Reply:
no it's not worth it then. just use `std::string` and value initialize it